tests/unit/lms/tasks/email_digests_test.py

```diff
@@ -223,148 +224,52 @@ def send_instructor_email_digest(self, patch):
-            ]
-        )
-
+    def test_it(self, created_after, created_before, digest_service, h_userid):
         send_instructor_email_digest(
             h_userid=h_userid,
+            created_after=created_after.isoformat(),
             created_before=created_before.isoformat(),
             override_to_email=sentinel.override_to_email,
         )
 
         digest_service.send_instructor_email_digest.assert_called_once_with(
-            h_userid,
-            datetime.fromisoformat(matching_task_done.data["created_before"]),
-            created_before,
-            override_to_email=sentinel.override_to_email,
-        )
-
-    def test_it_when_theres_no_matching_TaskDone(
-        self, created_before, digest_service, h_userid
-    ):
-        send_instructor_email_digest(
-            h_userid=h_userid, created_before=created_before.isoformat()
-        )
-
-        digest_service.send_instructor_email_digest.assert_called_once_with(
-            h_userid, created_before - timedelta(days=7), created_before
-        )
-
-    def test_it_when_TaskDone_is_more_than_a_week_old(
-        self, created_before, db_session, digest_service, h_userid, make_task_done
-    ):
-        db_session.add(
-            make_task_done(
-                created_before=(created_before - timedelta(days=8)).isoformat()
-            )
-        )
-
-        send_instructor_email_digest(
-            h_userid=h_userid, created_before=created_before.isoformat()
-        )
-
-        digest_service.send_instructor_email_digest.assert_called_once_with(
-            h_userid, created_before - timedelta(days=7), created_before
-        )
-
-    def test_it_when_TaskDone_doesnt_have_tzinfo(
-        self, created_before, db_session, digest_service, h_userid, make_task_done
-    ):
-        created_before = created_before.replace(tzinfo=None)
-
-        db_session.add(
-            make_task_done(
-                created_before=(created_before - timedelta(days=8)).isoformat()
-            )
-        )
-
-        send_instructor_email_digest(
-            h_userid=h_userid, created_before=created_before.isoformat()
-        )
-
-        digest_service.send_instructor_email_digest.assert_called_once_with(
-            h_userid=h_userid,
-            # The task adds tzinfo if it was not already present in the DB
-            created_after=created_before.replace(tzinfo=timezone.utc)
-            - timedelta(days=7),
+            h_userid="test_h_userid",
+            created_after=created_after,
             created_before=created_before,
-        )
-
-    def test_the_created_after_argument(self, created_before, digest_service, h_userid):
-        created_after = datetime(
-            year=2023, month=11, day=25, hour=5, tzinfo=timezone.utc
-        )
-
-        send_instructor_email_digest(
-            h_userid=h_userid,
-            created_before=created_before.isoformat(),
-            created_after=created_after.isoformat(),
```

Comment:
This was testing the optional argument. The argument is now non-optional and this is covered by the other tests above